### ISSUE TITLE
nginx proxypass port does not match netbox application port on 2.10.X

### ISSUE THREAD

**Issue by philipp1992** [bootc/netbox-chart#32]
Hi,

it seems that with the newer netbox version the post netbox listens on is not 8001 but 8080.
Since i dont see any way to tell netbox to run on a different port (e.g. via env-variable), the port of the nginx should be changed to 8080 or be made configurable.

configmap.yaml 

        location /{{ .Values.basePath }} {
          proxy_pass http://localhost**:8001;** -> should be 8080
          proxy_set_header X-Forwarded-Host $http_host;
          proxy_set_header X-Real-IP $remote_addr;
          proxy_set_header X-Forwarded-Proto $scheme;

this also affects the healthchecks of the netbox deployment.

kind regards
Philipp

**Comment by alop** [comment]
I see it as 8001
```
Operations to perform:
  Apply all migrations: admin, auth, circuits, contenttypes, dcim, extras, ipam, secrets, sessions, taggit, tenancy, users, virtualization
Running migrations:
  No migrations to apply.
💡 Superuser Username: admin, E-Mail: admin@example.com
↩️ Skipping startup scripts

979 static files copied to '/opt/netbox/netbox/static'.
✅ Initialisation is done.
[2021-02-02 20:43:21 +0000] [1] [DEBUG] Current configuration:
  config: /etc/netbox/config/gunicorn_config.py
  bind: ['0.0.0.0:8001']
```

**Comment by moonrail** [comment]
@alop 
https://github.com/netbox-community/netbox-docker has replaced tag v2.10.4 from a previous release with gunicorn, just as you've shown, to a new release with gunicorn replaced by nginx unit.

v2.10.3 is compatible to this chart, v2.10.4 (as of 2021/01/29) not anymore.
In netbox-docker releases, this would be anything from 1.0.0 onward.
https://github.com/netbox-community/netbox-docker/releases

This chart may requires adjustments in:
- Port of Netbox container in deployment from 8001 to 8080
- ProxyPass of Nginx in deployment from 8001 to 8080

Maybe even more, as logs show readOnly-FS errors for /opt/unit.

Maybe the Nginx container in deployment can be omitted completely.

**Comment by bootc** [comment]
This will be fixed with version 3.0.0 of the chart, which I'm currently working on in `master`. The fix for this particular issue is in https://github.com/bootc/netbox-chart/commit/38f4b943abdc353d1e8d13a83c98c1b03c2601db and is due to the switch from Gunicorn to NGINX Unit. Indeed that commit does away with the NGINX sidecar container.
